services/pul_kicker.md

```diff
@@ -0,0 +1,11 @@
+## File System Locks
+
+Occasionally servers will be locked down and we will need to kick them with a hard reboot. 
+
+The following steps will need a VPN connection
+
+* Using your preferred RDP Client ([Microsoft ships a good one](https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/remote-desktop-mac)) connect to `lib-staff998` -You will be prompted for you Active Directory netid and password.
+* On the resulting Desktop is a shortcut to the VSphere Application - Credentials are on lastpass-
+* Find the VM and restart or otherwise correct the mistake.
```

Comment:
The shortcut is VMware Vcenter.

Reply:
Clicking this shortcut takes you to `This site is not secure`
`Details -> Go on to the webpage` takes you a a different page where you need to select: `1.Vsphere webclient (flash)` or `2. vSphere client Html5 - partial functionality. `
Do we use on this step the credentials from Lastpass?

Reply:
Which one should be used? option 1 or option2 ?

Reply:
HTML5 one should be used.  The flash one does not work.

Reply:
Also I had to search for the VM, browsing showed me no machines.  That is OK becuase I was on to reboot a single machine, but I almost left without doing anything thinking I did not have access